src/main/groovy/org/pih/warehouse/inventory/CycleCountCandidateFilterCommand.groovy

```diff
@@ -0,0 +1,40 @@
+package org.pih.warehouse.inventory
+
+import grails.validation.Validateable
+import org.pih.warehouse.api.PaginationCommand
+import org.pih.warehouse.core.Tag
+import org.pih.warehouse.product.Category
+import org.pih.warehouse.product.ProductCatalog
+
+class CycleCountCandidateFilterCommand extends PaginationCommand implements Validateable {
```

Comment:
It's not achievable yet, because the cycle count table doesn't exist. Moreover I think we still need to determine how we would want to join the status data to this query. In my mind we would have to modify the view to join the cycle count table to return status. I would not recommend doing an additional query to cycle count table and then filtering it out statically, because we might lose the pagination.

Reply:
We discussed this in the meeting, but I agree that we should modify the view to join Cycle Counts, and also Cycle Count Requests. We likely need both because Request is the table that holds productId and so that's how we'll join with Candidate. And then we'll fetch the Cycle Count associated with that request to get the Cycle Count status (for the purpose of filtering out rows)

But yes that's for a future ticket